extensions/markdown-language-features/src/test/markdownLink.test.ts

```diff
@@ -83,50 +82,49 @@ suite('createEditAddingLinksForUriList', () => {
 	});
 
 	suite('pasteAsMarkdownLink', () => {
-
 		test('Should evaluate pasteAsMarkdownLink as true for selected plain text', () => {
-			const smartPaste = checkSmartPaste("hello! world", 0, 5);
+			const smartPaste = checkSmartPaste("hello! world", "hello! world", new vscode.Range(new vscode.Position(0, 5), new vscode.Position(0, 5)));
```

Comment:
To shorten code like this, you can write: `new Range(startLine, startCol, endLine, endCol)`. That way you don't also need to call `new Position`